ReleaseNotes/RN230316.md

```diff
@@ -0,0 +1,66 @@
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_ApplicationGateway_NetSec_Enable_WAF_Configuration](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#29-Azure_ApplicationGateway_NetSec_Enable_WAF_Configuration)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_LoadBalancer_NetSec_Restrict_Network_Traffic](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#30-Azure_LoadBalancer_NetSec_Restrict_Network_Traffic)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_Storage_AuthZ_Set_SAS_Expiry_Interval](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SASExpiryIntervalForStorageAccounts.ps1)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_FrontDoor_CDNProfile_NetSec_Enable_WAF_Configuration](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#32-Azure_FrontDoor_CDNProfile_NetSec_Enable_WAF_Configuration)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_SQLServer_AuthN_Dont_Allow_Public_Network_Access](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#41-Azure_SQLServer_AuthN_Dont_Allow_Public_Network_Access)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_SQLDatabase_AuthZ_Use_AAD_Only](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#39-Azure_SQLDatabase_AuthZ_Use_AAD_Only)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_AutomationAccounts_DP_Encrypt_Variables](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#40-Azure_AutomationAccounts_DP_Encrypt_Variables)
+
+## Tenant Scanner Framework Updates
+* Updated the target framework for all AzTS components to **net6.0** and function runtime version to **V4** in the entire AzTS solution as netcore3.1 is out of support.
```

Comment:
as **netcore 3.1** is out of support.

Pls. link phrase "out of support" with the Microsoft notice if available.